reposilite-frontend/src/components/Menu.vue

```diff
@@ -67,13 +64,24 @@ export default {
     methods: {
         handleLogout() {
             this.$store.dispatch('removeUser')
-            this.$parent.$parent.id = ''
+            this.$parent.id = ''
         }
     }
 }
 </script>
 
-<style lang="stylus" scoped>
+<style lang="stylus">
+.logo
+    font-family "Lucida Handwriting" !important
+    font-size 30px
+    margin-left 17px
+
 .navbar-burger
     height auto !important
+
+.navbar-item
+    font-family "Leelawadee UI"
+
+.navbar-end:last-child
+    padding-right 17px
```

Comment:
I think, this should be scoped. If any of these classes are created by buefy and you want to change their options then you can add separate style tag without `scoped`.